src/libmongoc/tests/test-mongoc-dns.c

```diff
@@ -643,7 +643,7 @@ _prose_loadbalanced_ping (mongoc_client_t *client)
    localhost.test.build.10gen.cc.
 */
 static void
-_prose_loadbalanced_run (bool pooled)
+_prose_test_9 (bool pooled)
```

Comment:
nit: ...on the other hand, is something like "_prose_test_9_loadbalanced_run" more clear? If I were looking for loadbalanced tests, "prose_test_9" probably wouldn't show up in my search. I don't have too strong of an opinion on this, though.

Reply:
I believe it more likely that one would search for this specific test by its prose test number (per SRV polling test specification) rather than by name. I believe the comments ("load balanced clusters") and referenced entities (`MONGOC_URI_LOADBALANCED`) should provide enough context for this test to be discoverable regardless.

Reply:
The consistency with the naming of prose tests from test-mongoc-crud.c seems like additional rationale for the name `_prose_test_9`. I am OK with either proposed name.

Reply:
Okay, sounds good!